Easydict/NewApp/View/SettingView/Tabs/GeneralTab.swift

```diff
@@ -136,6 +138,20 @@ struct GeneralTab: View {
                 Toggle(isOn: $hideMenuBarIcon) {
                     Text("hide_menu_bar_icon")
                 }
+                Picker(
+                    "modify_menubar_icon",
+                    selection: $selectedMenuBarIcon
+                ) {
+                    ForEach(MenuBarIconType.allCases) { option in
+                        if colorScheme == .light {
+                            Image(option.rawValue)
+                        } else {
+                            Image(option.rawValue)
+                                .renderingMode(.template)
+                                .foregroundStyle(.white)
```

Comment:
Can we use `.foregroundStyle(.primary)` here? So that we do not need a color schema here?